src/main/java/seedu/address/logic/parser/AddFeedCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.AddFeedCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.feed.Feed;
+
+/**
+ * Parses input arguments and creates a new AddFeedCommand object
+ */
+public class AddFeedCommandParser implements Parser<AddFeedCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddFeedCommand
+     * and returns an AddFeedCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddFeedCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_ADDRESS);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS)
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddFeedCommand.MESSAGE_USAGE));
+        }
+
+        String name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()).fullName;
+        String address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()).value;
```

Comment:
Will it be a huge hurdle to build a validation check before adding into the list? Personally believe that if the URL is wrong, then the application should not save it into the list in the first place.